deploy-manage/security/set-up-basic-security.md

```diff
@@ -1,23 +1,23 @@
 ---
-navigation_title: "Set up basic security"
+navigation_title: "Set up transport TLS"
 applies_to:
   deployment:
     self: ga
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/security-basic-setup.html
 ---
 
-
-
-# Set up basic security [security-basic-setup]
-
+% Scope: TLS certificates setup / multi-node cluster / manual configuration
+% original title: Set up basic security for the Elastic Stack
+# Set up transport TLS [security-basic-setup]
```

Comment:
consider x-links to the ECE/ECK/etc versions of these topics where they exist, or adding a note about "handled differently in other deployment types, see [link up to overview]"